containers/kubernetes/workloads/api-catalog-deployment.yaml

```diff
@@ -100,20 +100,20 @@ spec:
             - name: zowe-runtime
               mountPath: "/home/zowe/runtime"
             - name: zowe-config
-              mountPath: "/home/zowe/zowe.yaml"
+              mountPath: "/home/zowe/instance/zowe.yaml"
               subPath: zowe.yaml
               readOnly: true
             - name: zowe-keystore
               mountPath: "/home/zowe/keystore"
               readOnly: true
             - name: zowe-logs
-              mountPath: "/home/zowe/logs"
+              mountPath: "/home/zowe/instance/logs"
             - name: zowe-workspace
-              mountPath: "/home/zowe/workspace"
+              mountPath: "/home/zowe/instance/workspace"
       initContainers:
         - name: init-zowe
           # image: zowe-docker-release.jfrog.io/ompzowe/zowe-launch-scripts:2-ubuntu
-          image: zowe-docker-snapshot.jfrog.io/ompzowe/zowe-launch-scripts:2.0.0-ubuntu.v2-x-staging
+          image: zowe-docker-snapshot.jfrog.io/ompzowe/zowe-launch-scripts:2.0.0-ubuntu.users-jack-container-workspace-dir
```

Comment:
keeping this or going back to v2.x/staging?

Reply:
It's for testing purpose. Will be reverted before merge.